import/index_java/src/org/vufind/index/FormatCalculator.java

```diff
@@ -605,12 +605,20 @@ protected List<String> getFormatsFrom33xFields(Record record) {
             String desc = getSubfieldOrDefault(typeField, 'a', "");
             String code = getSubfieldOrDefault(typeField, 'b', "");
             String source = getSubfieldOrDefault(typeField, '2', "");
+            if (desc.equals("computer program")) {
```

Comment:
Is there a reason not to do the rdacontent check here? And, for that matter, if we do want to do the rdacontent check on everything, would it make sense to include everything inside a single if statement for RDA (or just do an early exit for non-RDA content)?

Reply:
My test cases use `rdacontent`, but I was thinking for this test that if any `336a` is "`computer program`" and the source is not `rdacontent`, it would still probably be better to rely on `getFormatFromRecordType()` to determine the format. Otherwise there is the risk of using `Video` or `VideoOnline` because of the next test.
That being said, I am not familiar with these values - do you know of relevant cases with non-RDA content ?